src/Component/Changenav.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { PureComponent } from 'react'
+import IntroComp from './IntroComp'
+import Counter from './CountClass'
+import AxiosDemo from './AxiosDemo'
+
+
+export default class Changenav extends PureComponent {
```

Comment:
as u have used pure component here , study difference between pureComponent and component